.gitignore

```diff
@@ -107,7 +107,7 @@ dist
 
 # doas binaries
 services/bin/doas
+services/bin/dropbear
 tools/doas/doas
 .enyocache
 dist/
-node_modules/
```

Comment:
Nice catch, that's a side-effect of working in a separate tree before final merging and not caring enough to figure out what was the previous gitignore contents...